cmd/gocritic/main.go

```diff
@@ -34,6 +35,11 @@ var subCommands = []*subCommand{
 		name:  "check-project",
 		short: "run gocritic over specified source tree, recursively",
 	},
+	{
+		main:  checkertools.NewMain,
+		name:  "checker-new",
```

Comment:
Not PR related, but: what about renaming commands to `package`, `project`, `newlinter` ?
Till we've some time before public release.

Reply:
I'm not sure. Can't alias solve problems for users that don't like typing extra `len("check-")`?